homeassistant/generated/integrations.json

```diff
@@ -5991,6 +5991,12 @@
       "config_flow": true,
       "iot_class": "cloud_push"
     },
+    "tvoverlay": {
+      "name": "TvOverlay",
+      "integration_type": "hub",
```

Comment:
Would `device` make more sense here, since you only connect to a single device per configuration entry?

https://developers.home-assistant.io/docs/creating_integration_manifest#integration-type

Reply:
Changed to `device`.

Reply:
hassfest check was failing with `integration_type:` `device`, changed `integration_type` to `service`.

Reply:
Looks like its still failing:

```
Integrations: 1196
Invalid integrations: 0

General errors:
* [CONFIG_FLOW] File integrations.json is not up to date. Run python3 -m script.hassfest

Error: Process completed with exit code 1.
```

when I run `python3 -m script.hassfest`, its changing `integration_type` back to `hub`.

Kindly suggest.

Reply:
Okay, my fault, you need to change this in the manifest.json. I,ve marked it on the wrong file.

Reply:
No worries, I have updated it in the manifest.json. Please have a look.